build/lib/i18n.js

```diff
@@ -533,12 +531,9 @@ function getResource(sourceFile) {
     else if (/^vs\/workbench/.test(sourceFile)) {
         return { name: 'vs/workbench', project: workbenchProject };
     }
-
-    // {{SQL CARBON EDIT}}
```

Comment:
>// {{SQL CARBON EDIT}} [](start = 4, length = 22)

why did you remove this line?

Reply:
this is a compiled file. It compiled and got rid of it